airbyte-integrations/connectors/source-retently/source_retently/schemas/feedback.json

```diff
@@ -0,0 +1,48 @@
+{
+  "type": "object",
+  "properties": {
+      "id": { "type": "string" },
+      "customerId": { "type": "string" },
+      "email": { "type": "string" },
+      "firstName": { "type": "string" },
+      "lastName": { "type": "string" },
+      "companyName": { "type": "string" },
+      "jobTitle": { "type": "string" },
+      "country": { "type": "string" },
+      "state": { "type": "string" },
+      "city": { "type": "string" },
+      "tags": {
+        "type": "array",
+        "items": { "type": "string" }
+      },
+      "customProps": {
+         "type": "array",
+         "items": { "type": "string" }
+       },
+      "campaignId": { "type": "string" },
+      "campaignName": { "type": "string" },
+      "createdDate": { "type": "string" },
+      "score": { "type": "number" },
+      "comment": { "type": "string" },
+      "checkbox": { "type": "boolean" },
+      "additionalQuestions": {
+        "type": "array",
+        "items": { "type": "string" }
```

Comment:
According to the API docs, this is not an array of string but an array of objects, with some properties nullable

```json
      "additionalQuestions": [
            {
                "type" : "RATING",
                "questionText" : "How likely are you to recommend our company to a friend or colleague?",
                "metricsType" : "NPS",
                "answer" : 10
            },
            {
                "type" : "MULTIPLE_OPTIONS",
                "questionText" : "What is the main reason for your score?",
                "answer" : "Support"
            },
        ],
```